lib/zinnia.js

```diff
@@ -94,6 +112,14 @@ export async function start ({
     const errorMsg = err instanceof Error ? err.message : '' + err
     const message = `Cannot start Zinnia: ${errorMsg}`
     onActivity({ type: 'error', message })
+    // We can get here in two ways:
+    // 1. `zinniad` cannot be started (execa returns error)
+    // 2. We don't receive any activity from `zinniad` before the timeout
+    // The first case can be easily handled by restarting() zinnia after a timeout,
+    // or maybe we should not do anything because most likely Zinnia is going to crash again.
+    // However, how to handle the timeout? We are leaving the process running, so eventually
+    // we will receive the first activity, then subsequent activities. All will work correctly,
+    // except the misleading error activity saying "Cannot start Zinnia".
```

Comment:
This is handled more elegantly in the `p-retry` PR, where any error thrown will lead to an abort and retry